Finish mobile version branch

ClintonYounge opened the pull request:
Added 4 sections for the 4 template projects.
Added the 5th section for "About Myself" information.

ClintonYounge committed:
Add html section for first displayed project

Added a section for the first project on the template.

ClintonYounge committed:
Add all necessary html code

Added all the necessary html code to the index.html file

ClintonYounge committed:
Add css for all project sections

Added all the necessary CSS for the projects sections.

ClintonYounge committed:
Update README.md file

Fixed error

ClintonYounge committed:
Minor edits to html and css

RaoAkif commented on style.css:
```diff
@@ -174,3 +174,525 @@ header nav div .menu-bar {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
```
![image](https://user-images.githubusercontent.com/61361037/212447770-b9407d23-879e-40d1-b14c-577509225a1a.png)

- In Figma `#C1C7D0` is used instead of `#344563`.
- Secondly, `#344563` color is being used 7 times in `style.css` file. Kindly use DRY code. But it in mind, replace it with `#344563` wherever FIGMA is using the other color.
To write DRY code, try not to repeat the same colors over and over again. To deal with repetition, you can define your color variables once, and then use them again. For Example:
```
  --color-white: #fff;
  --color-grey: #585858;
  --color-light-grey: #ebf0ee;
```

RaoAkif commented on index.html:
```diff
@@ -39,5 +39,189 @@ <h2 class="social order">LET'S CONNECT</h2>
+        <div class="top">
+          <h3 class="about-heading">About Myself</h3>
+          <p class="about-description">I'm a software developer! I can help you build a product, feature or website. Look
+            through some of my work and experience. If you like what you see and have a project you need coded, don't
+            hestiate to contact me.</p>
+          <h3 class="social order2">LET'S CONNECT</h3>
+          <ul class="social-links">
```
This is Figma design:
![image](https://user-images.githubusercontent.com/61361037/212447979-e8c301ff-c936-4902-8dff-f3ea7beac5a9.png)

This is your current design:
![image](https://user-images.githubusercontent.com/61361037/212448000-69e152da-052e-45f1-b26c-d9afa6d47139.png)

- In your design, there is a larger gap than the FIGMA design gap.
Try to be as closer to FIGMA design as you can.

RaoAkif commented on style.css:
```diff
@@ -174,3 +174,525 @@ header nav div .menu-bar {
+  flex-grow: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-items: flex-start;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  height: 72px;
+  margin-left: 0;
+}
```
Menubar Title & Dropdown arrow icon are not aligned as per FIGMA design.
Add `align-items: baseline;` to align the title of the menubar and dropdown arrow icon.
![image](https://user-images.githubusercontent.com/61361037/212448176-eab6a778-3b1e-496d-bcff-98c3269f2513.png)

RaoAkif commented on index.html:
```diff
@@ -39,5 +39,189 @@ <h2 class="social order">LET'S CONNECT</h2>
+          <p class="about-description">I'm a software developer! I can help you build a product, feature or website. Look
+            through some of my work and experience. If you like what you see and have a project you need coded, don't
+            hestiate to contact me.</p>
+          <h3 class="social order2">LET'S CONNECT</h3>
```
[OPTIONAL] For semantic purposes, if you have decided to use `h3` for the section title, for example for `About Myself`, then you can use `h4` or `h5` for that.

RaoAkif reviewed (changes requested):
## STATUS: REQUIRED CHANGES ♻️

Hi @ClintonYounge, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
_Check the comments under the review._
<img src="https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966" />

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ClintonYounge committed:
Update CSS

ClintonYounge committed:
Update html and css file

Updated the index.html and style.css file as per instructed by the code reviewer.

ClintonYounge committed:
Update index.html file

Basic edits to match Figma template

ClintonYounge committed:
Update style.css file

Fixed linter errors

ClintonYounge committed:
Update style.css file

Fixed final linter error

NduatiKagiri commented on index.html:
```diff
@@ -39,5 +36,190 @@ <h2 class="social order">LET'S CONNECT</h2>
+          <h2 class="social order">LET'S CONNECT</h2>
+
+          <ul class="social-links">
+            <li><a href="https://twitter.com/YoungeCjay" target="_blank"><i class="fa-brands fa-twitter li1"></i></a></li>
+            <li><a href="https://www.linkedin.com/in/clinton-younge-83386a25a/" target="_blank"><i class="fa-brands fa-linkedin li2"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-m li3"></i></a></li>
+            <li><a href="https://github.com/ClintonYounge" target="_blank"><i class="fa-brands fa-github li4"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-angellist li5"></i></a></li>
+          </ul>
```
- [ ] [ OPTIONAL ] Good job applying the font awesome icons, but kindly consider downloading the icons from the Figma design and using them in your project. This will help in having same-sized icons and avoid mixing up colors in the icons.

NduatiKagiri reviewed (approved):
Hi @ClintonYounge ,

# APPROVED 💯
 
Nice job implementing the requested changes. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![ea0cb5a48e3337523ffe8d60f177879d](https://user-images.githubusercontent.com/87394385/212485905-c4d5f286-6a37-4e02-9ecc-935920ff4f14.gif)

## Highlights ✔️
✅ GitHub Flow observed
✅ No Linters errors
✅ Smooth User Interface

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
